src/extensions/extension_manager/src/extension_manager.cpp

```diff
@@ -602,8 +602,7 @@ void ExtensionManager::_FreeComponentsDataString(char* componentsJson)
         goto done;
     }
 
-    if (ExtensionManager::_componentEnumeratorContractVersion.majorVer == ADUC_V1_CONTRACT_MAJOR_VER
```

Comment:
Did ADUC_ContractUtils_IsV1Contract exist already?

Reply:
Yes, it did exist. Should've been using it